skipchain/struct.go

```diff
@@ -522,6 +523,21 @@ func NewSkipBlockDB(db *bolt.DB, bn string) *SkipBlockDB {
 	}
 }
 
+// GetStatus is a function that returns the status report of the db.
+func (db *SkipBlockDB) GetStatus() onet.Status {
+	out := make(map[string]string)
+	db.DB.View(func(tx *bolt.Tx) error {
+		b := tx.Bucket([]byte(db.bucketName))
+		s := b.Stats()
+		out["Blocks"] = strconv.Itoa(s.KeyN)
```

Comment:
Why call it "Blocks"? Isn't this the number of keys?

Reply:
for skipchains it's the number of blocks.